frontend/src/components/footer/Footer.jsx

```diff
@@ -14,8 +14,6 @@ function Footer() {
             </div>
             <div className="footer-options-container">
                 <Link className="body-1-bold footer-option" to={routes.home}>About the Tool</Link>
```

Comment:
Can we get rid of the entire container? Pretty sure it looks weird with only one footer option

Reply:
I reverted the footer to its original state since I think Stacy is handling the changes in her PR.